iterative_refinement/expectation_maximization.py

```diff
@@ -115,12 +115,12 @@ def generate_xy_plane(n_pix):
         -------
         xy_plane : arr
             Array describing xy plane in space.
-            Shape (n_pix**2, 3)
+            Shape (n_pix, n_pix, 3)
         """
         # See how meshgrid and generate coordinates functions used
         # https://github.com/geoffwoollard/compSPI/blob/stash_simulate/src/simulate.py#L96
 
-        xy_plane = np.ones((n_pix**2, 3))
+        xy_plane = np.ones((n_pix, n_pix, 3))
```

Comment:
each pixel has three coordinates: xyz. I've been keeping the coordinates as shape `(n_pix*n_pix,3)`. For instance when you rotate each coordinate by a 3x3 rotation matrix, you just dot with the rotations.

I don't really see how the shapes would work well with the `xy_plane` being shape `(n_pix, n_pix, 3)`. Remember, it's not a scalar field. Each pixel has three numbers associated with it: xyz coordinates.